lib/ruby_ast_gen/ErbToRubyTransformer.rb

```diff
@@ -0,0 +1,115 @@
+require 'temple'
+require 'erb'
+
+class ErbToRubyTransformer
+  def initialize
+    @parser = Temple::ERB::Parser.new
+    @indent_level = 0
+    @current_line = []
+    @in_control_block = false
+    @control_block_content = []
+  end
+
+  def call(input)
```

Comment:
Might be more descriptive
```suggestion
  def transform(input)
```